crates/bevy_internal/src/default_plugins.rs

```diff
@@ -90,6 +90,11 @@ impl Plugin for IgnoreAmbiguitiesPlugin {
                 bevy_animation::advance_animations,
                 bevy_ui::ui_layout_system,
             );
+            app.ignore_ambiguity(
+                bevy_app::PostUpdate,
+                bevy_animation::animate_targets,
+                bevy_ui::ui_layout_system,
+            );
```

Comment:
I appreciate the previous context comments, that said I can understand they are difficult to balance between maintainability and correctness.

does that refer to `all external ambiguities silenced` ?

Reply:
This was justified using the comments just a few lines up, which state that UI cannot be animated :) I'd like to change that eventually of course!